changelog/index.php

```diff
@@ -37,28 +39,27 @@ protected function configure()
 	 */
 	protected function execute(InputInterface $input, OutputInterface $output)
 	{
-		// TODO iterate over all repos
-		$orgName = 'nextcloud';
-		$repoName = 'server';
-
-		$reposToIterate = [
-			"server",
-			"3rdparty",
-			"activity",
-			"apps",
-			"files_pdfviewer",
-			"files_texteditor",
-			"files_videoplayer",
-			"firstrunwizard",
-			"gallery",
-			"logreader",
-			"nextcloud_announcements",
-			"notifications",
-			"password_policy",
-			"serverinfo",
-			"survey_client",
-		];
-
+        $server = 'server';
+        $orgName = 'nextcloud';
+
+        // TODO iterate over all repos
+        $reposToIterate = [
+                "server",
+                "3rdparty",
+                "activity",
+                "apps",
+                "files_pdfviewer",
+                "files_texteditor",
+                "files_videoplayer",
+                "firstrunwizard",
+                "gallery",
+                "logreader",
+                "nextcloud_announcements",
+                "notifications",
+                "password_policy",
+                "serverinfo",
+                "survey_client",
+            ];
```

Comment:
Could you check that you used tabs with 4 spaces width? because this looks 🐟y.

Reply:
Sure. Please, check again.